tools/nvqpp/nvq++.in

```diff
@@ -499,7 +498,7 @@ for i in ${SRCS}; do
 done
 
 if ${DO_LINK}; then
-	run ${LINKER_CXX} ${LINKER_FLAGS} ${LINKDIRS} ${OBJS} ${LINKLIBS} ${OUTPUTOPTS}
+	run ${LINKER_CXX} ${LINKER_FLAGS} -Wl,-rpath,${install_dir}/lib -Wl,-rpath,${PWD} ${LINKDIRS} ${OBJS} ${LINKLIBS} ${OUTPUTOPTS}
```

Comment:
Hard coding these options in the final link step isn't the preferred solution.

Set a second LINKER_FLAGS variable to distinguish between the object relink step and the final executable link step.